workshops/operations.html

```diff
@@ -238,11 +238,20 @@
 	for details.
       </li>
       <li>
-	Groups that are doing setup and administration for workshops themselves
-	are not expected to pay this fee
+	Self-organised workshps are not expected to pay this fee
 	(but must still abide by
 	<a href="{{page.root}}/faq.html#trademark">these rules</a>
 	if they wish to use the Software Carpentry name and logo).
+        A workshp is considered to be <b>self-organised</b> if the group organising it
+        does all the setup and administration itself, in particular:
+       <ul>
+       <li>finds and communicates with the instructors and helpers without our administrative assistance,</li>
+       <li>sets up and manages participant registration,</li>
+       <li>handles workshop advertisement and provides us with sufficient information to add the workshop
```

Comment:
Instead of

> this is normally done by the lead instructor

what about

> normally send a link to one page similar to https://swcarpentry.github.io/workshop-template/, see instructions at https://github.com/swcarpentry/workshop-template

Could you split this into

- handles workshop advertisement
-